scipy/interpolate/_rgi.py

```diff
@@ -648,7 +688,7 @@ def interpn(points, values, xi, method="linear", bounds_error=True,
                                  "in dimension %d" % i)
 
     # perform interpolation
-    if method in ["linear", "nearest", "slinear", "cubic", "quintic", "pchip"]:
+    if method in ["linear", "nearest", "slinear", "cubic", "quintic", "pchip", "slinear_", "cubic_", "quintic_"]:
```

Comment:
Nitpick: Could we use a constant here (e.g. `_ALL_METHODS` from above?)

I'd normally use a set for this, but with such a few elements it really doesn't matter

Reply:
Good catch! Done.